cdap-ui/app/cdap/components/RulesEngineHome/RuleBooksTab/RulesBooksTab.scss

```diff
@@ -26,49 +51,61 @@ $create_rulebook_btn_font_color: #cccccc;
   .rule-books-container {
     display: flex;
     flex-wrap: wrap;
-    padding: 10px;
-    // Stretch the parent by 20px on right and left as children have 10px margin
-    // between them and its always 2 column layout
-    margin-left: -20px;
-    margin-right: -20px;
+    justify-content: space-between;
 
     .rule-book {
-      width: calc(50% - 20px);
+      margin: 15px 0 0;
+      height: 204px;
 
       &.center {
         display: flex;
         flex-direction: column;
-        align-items: center;
         border: 0;
         justify-content: center;
         padding: 0;
-        margin-top: 0;
-        margin-bottom: 0;
+        background-color: initial;
 
         &:only-child {
           width: 100%;
         }
 
         > div {
-          height: 50%;
+          min-height: 98px;
```

Comment:
It's specified in the specs here: https://issues.cask.co/secure/attachment/14268/RuleBooks%20Specs2.png